myhpi/search/templates/search/search_result.html

```diff
@@ -9,7 +9,7 @@ <h5 class="text-break">{{ result.title }}</h5>
                 {{ result.specific.date|date:"SHORT_DATE_FORMAT" }}
```

Comment:
*Adding comments to random lines to make them resolvable one by one*

Is it possible to remove the border (or make it same color as the highlight) on hover?

For that to work, we might need a more significant contrast between the hover highlight and the background color

This design:
![grafik](https://github.com/user-attachments/assets/c3874094-da9d-4a75-8c25-2569d1abebf2)

Old design:
![grafik](https://github.com/user-attachments/assets/1ac20aba-50e7-4474-ae05-18874cbaac5f)